### PR TITLE
feat(modelarts): add a new resource to get notebook flavors

### LINKED ISSUE #1881: [ModelArts]Add a data-source to query flavors of Notebook
Add a data-source to query flavors of Notebook , in order to create a Notebook instance
### Current Terraform Version

```
...
```

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #1881 

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
 make testacc TEST='./huaweicloud/services/acceptance/modelarts' TESTARGS='-run=TestAccDatasourceNotebookFlavors_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/modelarts -v -run=TestAccDatasourceNotebookFlavors_basic -timeout 360m -parallel 4
=== RUN   TestAccDatasourceNotebookFlavors_basic
=== PAUSE TestAccDatasourceNotebookFlavors_basic
=== CONT  TestAccDatasourceNotebookFlavors_basic
--- PASS: TestAccDatasourceNotebookFlavors_basic (8.44s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/modelarts 8.506s
```

### CODE REVIEW

File: docs/data-sources/modelarts_notebook_flavors.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+subcategory: "AI Development Platform (ModelArts)"
+---
+
+# huaweicloud_modelarts_notebook_flavors
+
+Use this data source to get available flavors of ModelArts notebook.
+
+## Example Usage
+
+```HCL
```
Comment (houpeng80) [inline docs/data-sources/modelarts_notebook_flavors.md]: change to lowcase

Comment (hmbig2) [reply docs/data-sources/modelarts_notebook_flavors.md]: done

File: docs/data-sources/modelarts_notebook_flavors.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+* `memory` - Memory size, in KB.
+
+* `vcpus` - Number of vCPUs.
+
+* `free` - Free flavor or not.
+
+* `sold_out` - Whether resources are sold out.
+
+* `billing` -
+  The [Billing](#notebookFlavors_FlavorsBilling) structure is documented below.
```
Comment (ShiChangkuo) [inline docs/data-sources/modelarts_notebook_flavors.md]: please merge into oneline

Comment (hmbig2) [reply docs/data-sources/modelarts_notebook_flavors.md]: done

**Pull request merged**